tox.ini

```diff
@@ -74,6 +77,13 @@ commands =
 deps = 
     -rrequirements-test.txt
 
+[testenv:unit_tests]
```

Comment:
Can you please explain why do we need a separate unit_tests env? It looks like this is doing the same as testenv:py38/39.
May I also ask if unit_tests should be mentioned above in envlist on line 4?

Reply:
Yes it is doing almost the same thing. The only difference is that `basepython` is not set for this one so it will take the one installed on the system. And for us it means it will pick up Python3.8 and Python3.9 as defined on line 17.
It is not mentioned in `envlist` so it is not automatically triggered when you run `tox` without parameters.

Reply:
Thank you for explaining. When I see the tox run for 3.8, I don't see the tests ran for python 3.8, not sure whats wrong there.
Also should we disable safety if it's failing out CI for some time?

Reply:
The run of on 3.8 was canceled because 3.9 failed. 
We can disable safety for now.